encoding.bs

```diff
@@ -152,6 +152,28 @@ run these steps:
  <li><p>Return <var>readItems</var>.
 </ol>
 
+<p>To <dfn for="I/O queue" export>peek</dfn> a number <var>number</var> of <a for=list>items</a>
```

Comment:
I keep going back and forth on whether we should keep the conversion algorithms and add a peek operation, or remove the conversions and add wait. See https://github.com/whatwg/encoding/pull/215#issuecomment-658836516

Reply:
I don't think we necessarily need to add conversions now. And we probably want to do something in Infra around conversions of these I suspect as they're generally useful.

Reply:
There are algorithms that use those conversions, and that before this PR weren't linked to any dfn. We might want to have a short paragraph defining how conversion works with consuming the queue and creating a queue with an EOF, and have a `class=XXX` note saying that Infra is expected to define infrastructure around conversions.